preprocessing/data_transformer.py

```diff
@@ -266,7 +266,7 @@ def clean_categorical_features(dataframe_dict: object,
     # Checking cycle
     if print_results:
         print('*' * 10)
-        print(f"Checking the difference in categorical columns in {dataframe_dict.keys()[0]} and {dataframe_dict.keys()[1]} datasets: ")
+        print(f"Checking the difference in categorical columns in {list(dataframe_dict.keys())[0]} and {list(dataframe_dict.keys())[1]} datasets: ")
```

Comment:
maybe for two keys, the list could not be a problem, but I would suggest following the best practice as Ahmed suggested to me before:
next(iter(dataframes_dict.keys()))

Reply:
For some reason the previous variant (without a list) doesn't for me at all any more.
just checked `next(iter(dataframe_dict.keys())` - it works - will replace to it

Reply:
replaced it with labels and iterator